.cc-metadata.yml

```diff
@@ -0,0 +1,8 @@
+# Whether this GitHub repo is engineering related
+engineering_project: true
+# Name of the repository/project in English
+english_name: cc-licenses
+# All technologies used
+technologies: Python, HTML
+# Whether this repository should be featured on the CC Open Source site
+featured: false
```

Comment:
please add newline at the end of the file.

Reply:
Made the suggested changes @kgodey